bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -19,7 +19,7 @@ <h2 id="security-vulnerability-bounty">Security Vulnerability Bounty</h2>
   <p>Mozilla will pay a bounty for client security bugs as detailed below. All security bugs must follow the following general criteria to be eligible:</p>
 
   <ul class="mzp-u-list-styled">
-    <li>Eligible security bugs may be present in any of the current main development or released versions of Firefox, Firefox for Android, or Firefox for iOS as released by Mozilla Corporation (e.g. Nightly mozilla-central or Beta test versions, as well as the final release product versions).</li>
+    <li>Eligible security bugs may be present in any of the main development or released versions of Firefox, Firefox for Android, or Firefox for iOS as released by Mozilla Corporation (e.g. Nightly mozilla-central or Beta test versions, as well as the final release product versions). Non-default configurations are <a href="{{ url('security.bug-bounty.faq') }}#nondefault-pref">not consistently eligible</a> and End-of-life products are not eligible for a security bug bounty.</li>
```

Comment:
I'm not sure about the word "consistently" here but I can live with it.  "not always eligible"? "only sometimes eligible"? Leave that part out entirely (since it's covered elsewhere) and get right to the main point "End-of-life products are not eligible for a security bug bounty." ?

Reply:
Went with "only sometimes" and reordered the sentence.